### PR TITLE
Completed prep-css-flexbox exercise. - Michael Suh

### PR DESCRIPTION
**Pull request by michael-suh**
display-flex
flex-wrap
justify-content
align-items

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: You'll want to make sure that `flex-basis` is used on your `.card-wrapper`. Some flexbox properties are used on the "flex container" (parent) and others are used on the "flex items" (children).

File: prep-css-flexbox/justify-content.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: You'll want to make sure that `flex-basis` is used on your `.card-wrapper`. Some flexbox properties are used on the "flex container" (parent) and others are used on the "flex items" (children).

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-flexbox/align-items.html (hunk 1 above)
Comment (michael-suh) [reply prep-css-flexbox/align-items.html]: align-items

File: prep-css-flexbox/justify-content.html (hunk 2 above)
Comment (michael-suh) [reply prep-css-flexbox/justify-content.html]: justify-content

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**